Vagrantfile

```diff
@@ -5,7 +5,7 @@ Vagrant.configure('2') do |c|
   server_post_up = <<-EOS.gsub(/^[[:blank:]]{4}/, '')
     Now run the following commands to get going:
 
-        simp config -a /vagrant/answers.yaml
+        simp config -a /vagrant/simp_config_answers.yaml
```

Comment:
Yeah, I saw that in the header.  If it's all the same, though, I think using something like this is better for distribution.  My goal here was to make it easier to read and understand, because this will be used for training and probably by users wanting to try out SIMP without too much work.